R/zchunk_LA115.roofPV.R

```diff
@@ -29,11 +29,39 @@ module_energy_LA115.roofPV_DISABLED <- function(command, ...) {
     # Load required inputs
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
     A15.roofPV_curves <- get_data(all_data, "energy/A15.roofPV_curves")
-    A15.roofPV_TechChange <- get_data(all_data, "energy/A15.roofPV_TechChange")
-    L100.Pop_thous_ctry_Yh <- get_data(all_data, "L100.Pop_thous_ctry_Yh")
+    A15.roofPV_TechChange <- get_data(all_data, "energy/A15.roofPV_TechChange") #not used
+    L100.Pop_thous_ctry_Yh <- get_data(all_data, "socioeconomics/L100.Pop_thous_ctry_Yh")
 
     # ===================================================
     # TRANSLATED PROCESSING CODE GOES HERE...
+
+    #Categorizing countries and regions to GCAM Region ID; selecting population in 2010
```

Comment:
Here and throughout, put a space between `#` and the start of the comment please